### ISSUE TITLE
AVT 1 - Carbon X Toggle Component DAP violations

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#1836]
## Environment
macOS Mojave version 10.14.2
Chrome version 72.0.3626.96 (Official Build) (64-bit)

## Detailed description
AVT 1 Run DAP on the example and there are 2 violations. See screenshot.
<img width="819" alt="screen shot 2019-02-15 at 8 48 40 am" src="https://user-images.githubusercontent.com/21676914/52863939-afc10600-30fe-11e9-885a-64d382a76819.png">

Note: To fix the violations add aria-disabled="true" to the example.

**Comment by snidersd** [comment]
Although the recommended fix will remove the violation it turns out that there is a DAP [GitHub issue 1184](https://github.ibm.com/IBMa/Tools-Rules-HTML/pull/1186) that is causing the violation in error and it should be fixed in the next DAP release expected later this month.  This should be re-tested with the new ruleset.

**Comment by snidersd** [comment]
@dakahn As per my comment Feb. 15, we found this to be a known DAP defect, please close.
